README.md

```diff
@@ -133,6 +133,76 @@ B F oz5xJ 67i1B 1B 7P zI ha xL 7Y
 * The second letter `F` is the header, corresponding to the precision 5 and no 3rd dimension set.
 * The rest of the string are the encoded coordinates.
 
+## Pseudocode
+
+The following pseudocode illustrates the steps needed to decode an encoded string and
+might also be a helpful template for an actual implementation.
+The main function to implement is taking the encoded coordinates (characters after
+the 2 header characters) as input and returns the sequence of signed integer values.
+
+```commandline
+function decode_coordinates_to_signed_values
```

Comment:
It does not decode coordinates, it decodes the encoded string into a list of signed deltas. Maybe rename it to match that better? e.g. `decode_signed_deltas` etc?